### PR TITLE
Backend yaml to startup the technolution/asm wrapper

### PR DESCRIPTION
**Pull request by Kleijwegt**

### CODE REVIEW

File: install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    role: mbsem,
+    children: ["ASM", "EBeamScanner", "MirrorDescanner", "MPPC",],
+}
+
+"ASM": {
+  class: technolution.AcquisitionServer,
+  role: "main",
+  init: {
+    # Use following host url for real system instead of simulator: "http://192.168.10.10:8080/v2"
+    host: "http://localhost:8080/v2",
```
Comment (Kleijwegt) [inline install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml:12]: Do we want as default the simulator or the real system url?

File: src/odemis/driver/technolution.py [hunk 2]
```diff
@@ -175,21 +175,21 @@ def __init__(self, name, role, host, children, externalStorage, **kwargs):
             ckwargs = children["EBeamScanner"]
         except Exception:
             raise ValueError("Required child EBeamScanner not provided")
-        self._ebeam_scanner = EBeamScanner(parent=self, **ckwargs)
+        self._ebeam_scanner = EBeamScanner(parent=self, daemon=kwargs.get("daemon"), **ckwargs)
```
Comment (Kleijwegt) [inline src/odemis/driver/technolution.py:178]: Changes made to to the technolution wrapper to have the yaml startup the wrapper correctly.

File: install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml (hunk 1 above)
Comment (pieleric) [reply install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml:12]: For such file, it'd be the real system IP.

**Review by pieleric — APPROVED** [review]

File: install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml (hunk 1 above)
Comment (Sabrina85) [reply install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml:12]: I agree it should be the real system IP as this yaml file is placed in the hwtest folder. All simulator yaml files are in the sim folder.

File: install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+"EBeamScanner": {
+  # Internal child of ASM, so no class
+  role: "multibeam",
+}
+
+"MirrorDescanner": {
+  # Internal child of ASM, so no class
+  role: "galvo",
```
Comment (Sabrina85) [inline install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml:31]: "galvo" does not look like a good name to me. The corresponding child is named "MirrorDescanner". It is not obvious that the mirror descanner is a mirror galvonometer. I suggest some roles that are inspired by the class names as we usually stick with them "forever". "mppc" and "multibeam" are fine, for the galvo I suggest maybe "multibeam-descanner" or just "descanner" to keep it short?

**Review by Sabrina85 — CHANGES_REQUESTED** [review]

File: install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml (hunk 3 above)
Comment (Kleijwegt) [reply install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml:31]: Let's go for descanner then.

**Review by Sabrina85 — APPROVED** [review]
Looks good!

File: install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+MultiBeamSEM: {
+    class: Microscope,
+    role: mbsem,
+    children: ["ASM", "EBeamScanner", "MirrorDescanner", "MPPC",],
+}
+
+"ASM": {
+  class: technolution.AcquisitionServer,
+  role: "main",
```
Comment (pieleric) [inline install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml:9]: If I understand correctly, this component is also useful for the client, right? (eg, to change the external storage). So it should indeed have a role. However, "main" doesn't sound good. What's the role that will be used for the fastem? Maybe "asm" would be fine?

Comment (Sabrina85) [reply install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml:9]: Indeed overlook this one. "asm" is good.

Comment (Kleijwegt) [reply install/linux/usr/share/odemis/hwtest/technolution-asm.odm.yaml]: Ok great, @pieleric I think it can be merged then.

**Pull request merged**